main.pm

```diff
@@ -503,6 +503,7 @@ sub load_consoletests() {
         }
         loadtest "console/mtab.pm";
         if (!get_var("NOINSTALL") && !get_var("LIVETEST") && (check_var("DESKTOP", "textmode"))) {
+            loadtest "console/yast2_nfs_server.pm";
```

Comment:
It's quicker than the other two tests below (mysql and HTTP) and has similar scope/importance IMO. There's also only one test with DESKTOP=textmode AFAICS.